pigeon/notification.py

```diff
@@ -0,0 +1,22 @@
+class Notification:
+    """
+    Notification holds the information necessary to notify a user.
+
+    Notifying a user requires at least one notification handler to be
+    registered using `register_handler`.
+    """
+    handlers = {}
+
+    def __init__(self, about):
+        """Store the object the notification is about."""
+        self.about = about
```

Comment:
I don't think that it's fair to prescribe that the notification will be about anything in particular.

For example, what would a "Welcome" email be "about"?